core/src/actions/properties.ts

```diff
@@ -371,12 +371,13 @@ export class PropertyRecordPreview extends AuthenticatedAction {
     let newPropertyValues: Array<string | number | boolean | Date> = [];
     let errorMessage: string;
     try {
-      newPropertyValues = await source.importRecordProperty(
-        record,
-        property,
-        params.options,
-        params.filters
-      );
+      newPropertyValues =
+        (await source.importRecordProperty(
+          record,
+          property,
+          params.options,
+          params.filters
+        )) ?? [];
```

Comment:
Adding some more saftey around a preview not finding any values - this and `ui/ui-components/components/property/RecordPreview.tsx` could crash without any values returned prior to this